apps/io-func-wallet-solution-backend/src/app/main.ts

```diff
@@ -58,3 +66,10 @@ app.http("getEntityConfiguration", {
     signer,
   }),
 });
+
+app.http("getNonce", {
+  methods: ["GET"],
+  authLevel: "anonymous",
```

Comment:
why anonymous?

Reply:
I think it's a typo from the old configuration where they were all `anonymous` since the endpoints were directly accessible by anyone. Since here I believe we will have at least an APIM or a proxy we can change in `function`

Reply:
I put `anonymous` like the other functions because I thought we would discuss authentication later. Should I put `function` here?

Reply:
we can leave anonymouse, then we'll fix it with another PR later

Reply:
We thought about it more carefully. In fact, the nonce in the DB have a TTL in the order of minutes, therefore the probability that a random string is equal to one generated a few minutes ago is so low that we can ignore this. So I would say not to add the retry and still leave an error in case the insert is not successful.